notifications/views.py

```diff
@@ -163,3 +174,51 @@ def live_unread_notification_list(request):
         'unread_list': unread_list
     }
     return JsonResponse(data)
+
+
+def live_notification_list(request):
+    if not request.user.is_authenticated():
+        data = {
+           'all_count':0,
+           'all_list':[]
+        }
+        return JsonResponse(data)
+
+    try:
+        num_to_fetch = request.GET.get('max', 5)  # If they don't specify, make it 5.
```

Comment:
this will feel a lot simpler if you combine it all into 1 line instead of 4:

`min(max(1, int(request.GET.get('max', 5))), 100)`